external/webserver/http_client.c

```diff
@@ -950,17 +957,21 @@ static int http_send_chunk_buffer(struct http_client_t *client, char *buf)
 	return 0;
 }
 
-static int http_send_chunk(struct http_client_t *client, char *buf, int buflen, const char *body, bool is_last_msg)
+static int http_send_chunk(struct http_client_t *client, char *buf, int buflen,
+					const char *body, int body_len, bool is_last_msg)
 {
 	int ret = 0;
+	int buf_len = 0;
 
-	ret = prepare_chunk_body(buf, buflen, body, is_last_msg);
+	ret = prepare_chunk_body(buf, buflen, body, body_len, is_last_msg);
```

Comment:
of course prepare_chunk_body & http_Send_chunk_buffer checking it but to avoid execute useless code flow, null check for client & buf  should be done here...

Reply:
done